app/jobs/scheduled/tl3_promotions.rb

```diff
@@ -22,13 +22,30 @@ def execute(args)
             # Don't demote too soon after being promoted
             next if u.on_tl3_grace_period?
 
+            demoted_user_id, modifier_applied =
```

Comment:
What feels odd to me here is the argument order vs the return value order. `false, u, demoted_user_ids`, but return is `demoted_user_id, modifier_applied`